src/index.css

```diff
@@ -150,6 +150,10 @@ body {
   content: '';
 }
 
+.textBreakWords {
+  word-break: break-word;
```

Comment:
We don't need this: https://tailwindcss.com/docs/word-break

Reply:
`overflow-wrap: break-word'` works like `word-break: break-word` as I know

Reply:
I tried but it didn't work as I expected, so I decided to add the right css property that will fix the issue.